libcloud/compute/drivers/openstack.py

```diff
@@ -4813,3 +4910,47 @@ def __repr__(self):
             ' snapshots="%s", backups="%s">'
             % (self.volumes, self.gigabytes, self.snapshots, self.backups)
         )
+
+
+class OpenStack_2_ServerGroup(object):
+    """
+    Server Group info.
+
+    See:
+    https://docs.openstack.org/api-ref/compute/?expanded=create-server-detail,list-server-groups-detail#server-groups-os-server-groups
+    """
+
+    def __init__(
+        self,
+        id,
+        name,
+        policy,
+        members=[],
```

Comment:
Should avoid mutable defaults here for ``members`` and ``rules``argument and default to ``None`` and set the value to empty list if it's not set inside the constructor (self.members = members or []``.